### DISCUSSION TITLE
A better tooltip for `Find and replace`

### DISCUSSION THREAD

**Discussion by macmarrum** [freeplane/freeplane#454] [Ideas]
`Edit->Find->Find and replace...` opens the window

![Find_and_replace](https://user-images.githubusercontent.com/7197131/173521600-9cbae585-e0a8-4f4c-8530-eba0bf568512.png)

The list of nodes displayed in the window is limited by the currently applied filters, i.e. only visible nodes are listed.

The tooltip for the actions says
`Shows all nodes as a searchable list with filter properties.`

In my opinon, a better tooltip would be
`Shows visible (filtered) nodes as a searchable list.`

NB. I am not sure what was originally meant by `with filter properties` and I fail to understand its meaning. Given that the searchable list contains only visible nodes, i.e. after applying filters, the part `with filter properties` might introduce confusion.

Any thought on the proposed tooltip text?

**Comment by rexelbartolome** [comment]
I understand the proposed tooltip much better than the original one. I don't understand what `filter properties` mean too so I'm on board with the change :)

**Comment by dpolivaev** [comment]
Preview 1.10.3_16

**Comment by ldmpub** [comment]
Thanks, translation improved in french.

Note: from my point of view the window does not contain a `list `but rather a `table `=> I performed the change for french, you may do the same for english? ...

Direct link for this string in Weblate:
https://hosted.weblate.org/translate/freeplane/translation/en/?checksum=62ee8f4f4c8d9d88
